go-controller/pkg/clustermanager/default_network_cluster_controller.go

```diff
@@ -0,0 +1,392 @@
+package clustermanager
+
+import (
+	"fmt"
+	"net"
+	"reflect"
+	"sync"
+
+	corev1 "k8s.io/api/core/v1"
+	cache "k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+
+	hotypes "github.com/ovn-org/ovn-kubernetes/go-controller/hybrid-overlay/pkg/types"
+	houtil "github.com/ovn-org/ovn-kubernetes/go-controller/hybrid-overlay/pkg/util"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/clustermanager/subnetallocator"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/factory"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kube"
+	objretry "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/retry"
+	ovntypes "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+type networkClusterControllerBase struct {
+	kube         kube.Interface
+	watchFactory *factory.WatchFactory
+	stopChan     chan struct{}
+	wg           *sync.WaitGroup
+
+	// node events factory handler
+	nodeHandler *factory.Handler
+
+	// retry framework for nodes
+	retryNodes *objretry.RetryFramework
+}
+
+// defaultNetworkClusterController is the cluster controller for the default network
+// for all the nodes. Listens to the node events and allocates subnet from the
+// default cluster subnet pool.
+type defaultNetworkClusterController struct {
+	networkClusterControllerBase
+	nodeNetworker *NodeNetworker
```

Comment:
hrrrmmm not convinced with the new name haha

What about `nodeSubnetManager` or `nodeNetworkManager`?

Reply:
OK, seeing the additional comment from @trozet, I am considering further renames:

`secondaryNetworkClusterControllerManager` -> `secondaryNetworkClusterManager`
`NodeNetworker` -> `clusterNetworkManager`

WDYT?

Reply:
secondaryNetworkClusterControllerManager -> secondaryNetworkClusterManager  :  Sounds good to me.

I have renamed NodeNetworker -> NodeNetworkManager.  Does that sound good ?  Or  you'd prefer clusterNetworkManager ?

Reply:
I made the `clusterNetworkManager` suggestion based on @trozet opinion that having the work `node` in the name might be confused as being stuff related to node mode instead of cluster manager mode. I think  `clusterNetworkManager` would avoid that.